pysat/instruments/pysat_testing2d.py

```diff
@@ -30,7 +30,7 @@ def load(fnames, tag=None, sat_id=None):
     month = int(parts[-3])
     day = int(parts[-2])
     date = pysat.datetime(yr,month,day)
-    scalar = 50
+    scalar = 100
```

Comment:
Why?  A comment with some explaination about the choice of scalar (even "arbitrary scalar value for testing") would be helpful here.

Reply:
Indeed. Will add it.

Reply:
Commit pushed.